tests/publiccloud/instance_overview.pm

```diff
@@ -55,11 +55,23 @@ sub run {
         zypper_call("in bzip2");
     }
 
-    assert_script_run("rpm -qa > /tmp/rpm.list.txt");
-    upload_logs('/tmp/rpm.list.txt', timeout => 180, failok => 1);
-
     assert_script_run("SUSEConnect --status-text", 300);
     zypper_call("lr -d");
+
+    collect_system_information();
+}
+
+sub collect_system_information {
+    # Collect various system information and pack them to instance_overview.tar
+    script_run("cd /var/tmp");
+    assert_script_run("mkdir -p instance_overview");
+    assert_script_run("rpm -qa | tee instance_overview/rpm.list.txt", timeout => 90);
+    assert_script_run("cat /proc/cpuinfo | tee instance_overview/cpuinfo.txt");
+    assert_script_run("cat /proc/meminfo | tee instance_overview/meminfo.txt");
+    assert_script_run("uname -a | tee instance_overview/uname.txt");
+    assert_script_run("tar -xcf instance_overview.tar.gz instance_overview/");
```

Comment:
FYI there is a `tar_and_upload_log`. i leave it here just in case you like it more

Reply:
> FYI there is a `tar_and_upload_log`. i leave it here just in case you like it more

That looks cool!

Reply:
Good idea! Gonna try to implement that.

Reply:
This is nice. Done.